cli/src/commands/wallet.rs

```diff
@@ -247,15 +247,108 @@ pub mod delete_command {
     }
 }
 
+pub mod export_command {
+    use super::*;
+
+    command!(CommandMetadata::build("export", "Export opened wallet to the file")
+                .add_required_param("export_path", "Path to the export file")
+                .add_required_deferred_param("export_key", "Passphrase used to derive export key")
+                .add_example("wallet export export_path=/home/indy/export_wallet export_key=key")
+                .finalize()
+    );
+
+    fn execute(ctx: &CommandContext, params: &CommandParams) -> Result<(), ()> {
+        trace!("execute >> ctx {:?} params {:?}", ctx, params);
+
+        let (wallet_handle, wallet_name) = ensure_opened_wallet(&ctx)?;
+
+        let export_path = get_str_param("export_path", params).map_err(error_err!())?;
+        let export_key = get_str_param("export_key", params).map_err(error_err!())?;
+
+        let export_config: String = json!({ "path": export_path.clone(), "key": export_key.clone() }).to_string();
+
+        trace!("Wallet::export_wallet try: wallet_name {}, export_path {}, export_key {}", wallet_name, export_path, export_key);
```

Comment:
I am not sure if it is good idea to log export key?

Reply:
Yeah it is a bad idea. I believe we have this logging problem not only in this place.